code/modules/paperwork/records/info/waw.dm

```diff
@@ -3,15 +3,17 @@
 	name = "T-01-114"
 	info = {"<h1><center>T-01-114</center></h1>	<br>
 	Name : The Warden	<br>
-	Risk Class : Waw	<br>
+	Risk Class : WAW	<br>
 	Max PE Boxes : 24	<br>
 	Qliphoth Counter : 2	<br>
 	Work Damage Type : Black	<br>
 	Work Damage : Moderate	<br>
-	- When neither Fortitude nor Justice were 4 or higher, the Qliphoth Counter lowered.	<br>
+	- When employees of Justice Level 3 or lower finished working with The Warden, the Qliphoth Counter lowered.	<br>
```

Comment:
It's not an "Or" it's an "And". Both have to be below level 4.

Reply:
I know, but the way it's written is "If your justice is 3 or lower, Q lowered." and "If your fort is 3 or lower, Q lowered." 
It should be  "If your fort and justice are both 3 or lower, Q lowered"